app/controllers/junior_stories_controller.rb

```diff
@@ -33,4 +37,41 @@ def junior_story_params
       :remote, :tech_team_size, :company_size, :company_age, :person_of_colour, :other,
       :publishing_consent, :freelancer)
   end
+
+  def allowed_params
```

Comment:
these values are exactly the same as the `junior_story_params` above, aren't they - what's the benefit of doing it this way?